scripts/build/package.js

```diff
@@ -27,7 +27,7 @@ const indexTemplate = fs.readFileSync(indexTemplateFile, UTF8);
 const iconObjectTemplate = fs.readFileSync(iconObjectTemplateFile, UTF8);
 
 const data = require(dataFile);
-const { getIconSlug, titleToSlug } = require("../utils.js");
+const { getIconSlug, svgToPath, titleToSlug } = require("../utils.js");
```

Comment:
Looks good, but I think this should be a [local helper function](https://github.com/simple-icons/simple-icons/blob/b9cc1788d1efa38de69a17e1c8cb9a86eeed4954/scripts/build/package.js#L32) based on https://github.com/simple-icons/simple-icons/pull/6189#discussion_r693525489.

Reply:
🤔  I'm not really sure, perhaps `svgToPath` makes more sense to declare it an utility as it is more general than `slugToVariableName` because is converting from an attribute to another like `titleToSlug`. Maybe @ericcornelissen could clarify here what he relies on to declare functions as utilities or locals ones.

Reply:
As I said in the other comment, it's not very clear where these kinds of functions should go at the moment. It's mostly a matter of "what feels right". I tend to ask myself the question: "could this be useful outside of this script?"

I would agree with @mondeja here that it's similar to `titleToSlug` in that converting between these properties may be useful elsewhere as well, and therefor can stay in the utilities file.

Reply:
> could this be useful outside of this script?

Oh okay, it makes sense to put it in the utilities file then.